pkg/apis/tensorflow/v1alpha1/defaults.go

```diff
@@ -0,0 +1,77 @@
+package v1alpha1
+
+import (
+	"fmt"
+	"github.com/golang/protobuf/proto"
+	"github.com/tensorflow/k8s/pkg/util"
+	"k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+)
+
+
+func addDefaultingFuncs(scheme *runtime.Scheme) error {
+	return RegisterDefaults(scheme)
+}
+
+
+// SetDefaults sets any unspecified values to defaults
```

Comment:
fix this lints

Reply:
Can you attach more info about lints errors :)
It maybe helpful to fix this.